app/src/main/res/xml/appfilter.xml

```diff
@@ -468,7 +481,6 @@
     <item component="ComponentInfo{com.samsung.android.contacts/com.samsung.android.contacts.contactslist.PeopleActivity}" drawable="contacts" />
     <item component="ComponentInfo{com.sonyericsson.android.socialphonebook/com.sonyericsson.android.socialphonebook.LaunchActivity}" drawable="contacts" />
     <item component="ComponentInfo{com.sonyericsson.android.socialphonebook/com.sonyericsson.android.socialphonebook.DialerEntryActivity}" drawable="contacts" />
-    <item component="ComponentInfo{com.google.android.contacts/com.android.contacts.activities.PeopleActivity}" drawable="contacts" />
```

Comment:
Just curious, why was this one removed?

Reply:
I just noticed it was to fix the Google Contacts issue.

Reply:
As mentioned in the commit message of e8cc47163f05af14680574fce02010637c482d04 and dkanada/frost#246 

This occurence is a duplicate of line 465 (now 478)